.github/scripts/create-lambda-function.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -euo pipefail
+
+LANG="$1"
+PR_NUMBER="$2"
+FUNCTION_NAME="$3"
+REGISTRY="$4"
+IMAGE="$5"
+ROLE_ARN="$6"
+
+FULL_FUNCTION_NAME="${FUNCTION_NAME}-${PR_NUMBER}-${LANG}"
```

Comment:
I don't think you need a LANG in this case?
```suggestion
FULL_FUNCTION_NAME="${FUNCTION_NAME}-${PR_NUMBER}"
```